src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -297,6 +297,17 @@ private function createFirewalls(array $config, ContainerBuilder $container)
         $mapDef->replaceArgument(0, new Reference('security.firewall.context_locator'));
         $mapDef->replaceArgument(1, new IteratorArgument($map));
 
+        if (!$this->authenticatorManagerEnabled) {
+            // add authentication providers to authentication manager
+            $authenticationProviders = array_map(function ($id) {
+                return new Reference($id);
+            }, array_values(array_unique($authenticationProviders)));
+
+            $container
+                ->getDefinition('security.authentication.manager')
+                ->replaceArgument(0, new IteratorArgument($authenticationProviders));
+        }
+
```

Comment:
```suggestion
```

Since Symfony 6.0, we only have the authenticator manager

Reply:
Thanks for your quick response :D It is fixed, sorry the false positive, it was a rebasing error.